lib/command/config.rb

```diff
@@ -2,9 +2,27 @@
 
 module Command
   class Config < Base
+    NAME = "config"
+    OPTIONS = [
+      app_option
+    ].freeze
+    DESCRIPTION = "Displays current configs (global and app-specific)"
+    LONG_DESCRIPTION = <<~HEREDOC
+      - Displays current configs (global and app-specific)
+    HEREDOC
+    EXAMPLES = <<~HEREDOC
+      ```sh
+      # Shows the global config.
+      cpl config
+
+      # Shows both global and app-specific configs.
+      cpl config -a $APP_NAME
+      ```
+    HEREDOC
+
     def call
       puts "-- Options"
-      puts config.options.to_yaml[4..]
+      puts config.options.to_hash.to_yaml[4..]
```

Comment:
`config.options` now comes from Thor and is a `HashWithIndifferentAccess`, so we convert it to a normal hash here.